index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="port-folio/main.css">
+    <title>Arksel's Portfolio</title>
+</head>
+<body>
+    <div class="mobile-container">
+            <nav class="navbar">
+                <div class="header">Logo</div>
+
+                <div class="menu">
+                    <button type="button" class="menu-btn"></button>
+                    <button type="button" class="menu-btn"></button>
+                    <button type="button" class="menu-btn"></button>
+                </div>
+
+            </nav>
```

Comment:
- [ ] Please enclose this block in a `header` tag 🙏 . It is because it follows HTML best practices